ruby-challenges.rb

```diff
@@ -10,6 +10,12 @@
 letter_t = 't'
 # Expected output: ['tea', 'water', 'soda water']
 
+def filter_words(array_of_words, letter)
+    array_of_words.select { |word| word.include?(letter.upcase) || word.include?(letter.downcase)}
+end
+
```

Comment:
Nice job. Casing is definitely a consideration. You may be able to upcase or downcase both sides of your evaluation and save yourself a bit of code. `word.downcase.include?(letter.downcase)`